lib/views/pay/pay_view.dart

```diff
@@ -17,18 +17,14 @@ class _PayViewState extends State<PayView> {
   String? boltString;
   int? amountMsat;
   AppPayInvoice? paymentResponse;
+  String? error;
 
   Future<AppPayInvoice> payInvoice(String boltString, int? amountMsat) async {
-    try {
-      final response = await widget.provider
-          .get<AppApi>()
-          .payInvoice(invoice: boltString, msat: amountMsat);
-      return response;
-    } catch (ex) {
-      AppPayInvoice error =
-          AppPayInvoice(payResponse: {"Error": ex.toString()});
-      return error;
-    }
+    // we don't need error handling here
+    final response = await widget.provider
+        .get<AppApi>()
+        .payInvoice(invoice: boltString, msat: amountMsat);
+    return response;
```

Comment:
We did needed an error handling there, the pay command can return a message error. Should've fixed it earlier. My bad.

Reply:
we need to rework all our error handling because we do not have one? :)

Maybe we can work with from the following issue? https://github.com/dart-lightning/lndart.clnapp/issues/89

Reply:
Sure